packages/core/src/components/ToggleButtonController/index.tsx

```diff
@@ -76,6 +77,7 @@ export default class ToggleButtonController extends React.Component<Props, State
         disabled={this.props.disabled}
```

Comment:
Can you destructure these also while you're here.

Reply:
sure!